tests/test_violations_analysis.py

```diff
@@ -114,22 +173,41 @@ def test_branch_loading(self) -> None:
         for args in branch_args:
             branch = Branch(*args)
             branch.disable()
-        self.assertEqual(
-            Violations.BRANCH_LOADING, check_violations(max_trafo_loading_pct=115.0)
-        )
-        self.assertEqual(
-            {100.0: ({4: [115.15924072265625]})},
-            ViolationsStats.asdict()[Violations.BRANCH_LOADING],
-        )
+        if sys.platform == "win32" and not envs.pandapower_backend:
+            self.assertEqual(
+                Violations.BRANCH_LOADING, check_violations(max_trafo_loading_pct=115.0)
+            )
+            self.assertEqual(
+                {100.0: ({4: [115.15924072265625]})},
+                ViolationsStats.asdict()[Violations.BRANCH_LOADING],
+            )
+        else:
+            print(ViolationsStats.asdict()[Violations.BRANCH_LOADING])
+            self.assertEqual(
+                Violations.BRANCH_LOADING,
+                check_violations(max_trafo_loading_pct=125.0, min_bus_voltage_pu=0.8),
+            )
+            self.assertEqual(
+                {100.0: ({5: [116.57410137972397]})},
+                ViolationsStats.asdict()[Violations.BRANCH_LOADING],
+            )
 
     def test_2w_trafo_loading(self) -> None:
         wf.open_case(DEFAULT_CASE)
-        self.assertEqual(Violations.TRAFO_LOADING, check_violations())
-        self.assertEqual(
-            {100.0: {6: [102.952880859375]}},
-            ViolationsStats.asdict()[Violations.TRAFO_LOADING],
-        )
+        if sys.platform == "win32" and not envs.pandapower_backend:
+            self.assertEqual(Violations.TRAFO_LOADING, check_violations())
+            self.assertEqual(
+                {100.0: {6: [102.952880859375]}},
+                ViolationsStats.asdict()[Violations.TRAFO_LOADING],
+            )
+        else:
+            self.assertEqual(Violations.TRAFO_LOADING, check_violations())
+            self.assertEqual(
+                {100.0: {6: [107.74284564774533]}},
```

Comment:
The violated values (`102.9` and `107.7`) could be put to the conditional assignment so that the test code isn't duplicated.